src/main/java/nebula/plugin/metrics/MetricsPluginExtension.java

```diff
@@ -121,8 +128,41 @@ public void setSanitizedProperties(List<String> sanitizedProperties) {
         this.sanitizedProperties = checkNotNull(sanitizedProperties);
     }
 
+    public String getSuroLogEventName() {
```

Comment:
That's a good idea. Maybe we could have the extension contain two internal classes, such that configuration would be something like:
```groovy
metrics.suro {
    hostname = ''
}

or metrics.elasticsearch {
   hostname = ''
}
```